src/app/view/templates/beol/beol.component.ts

```diff
@@ -154,15 +154,15 @@ export class BeolComponent implements OnInit {
 
         const Du_Fresne = new Correspondent("Du Fresne", "(DE-588)1142274934");
 
-        const Amedee_Francois_Frezier = new Correspondent("Amedee Francois Frezier", "(DE-588)117537373");
+        const Amedee_Francois_Frezier = new Correspondent("Amedee François Frezier", "(DE-588)117537373");
 
-        const S_J_Vitus_Friesl = new Correspondent("S J Friesl", "(DE-588)10012786X");
+        const S_J_Vitus_Friesl = new Correspondent("S J Vitus Friesl", "(DE-588)10012786X");
```

Comment:
actually the first name does not have S J
name corresponding to this gnd number in both  nameindex and dnb is: Vitus, Friesl
https://portal.dnb.de/opac.htm?query=10012786X&method=simpleSearch&cqlMode=true


--

Reply:
done in https://github.com/dhlab-basel/Salsah/pull/206/commits/a1e7e6768121de38743b51670be2e1168d3b6395